server/Makefile

```diff
@@ -4,7 +4,7 @@ SCHEMATYPER := github.com/idubinskiy/schematyper
 MANIFEST_DIR := pkg/plugin/manifest
 
 DATALOADEN := github.com/vektah/dataloaden
-GQL_PACKAGE := github.com/reearth/reearth-visualizer/server/internal/adapter/gql
+GQL_PACKAGE := github.com/reearth/reearth/server/internal/adapter/gql
```

Comment:
[memo]
module name of go.mod is `github.com/reearth/reearth/server`, but this package name is different.
so `dataloaden` can't refer correct model.

Reply:
@pyshx 
we can generate dataloader through go generate.
https://github.com/reearth/reearth-visualizer/blob/main/server/internal/adapter/gql/gqldataloader/dataloader.go

but Makefile command was not able to apply it, so I added support for make gql to generate it on this [commit](https://github.com/reearth/reearth-visualizer/pull/1332/commits/9b3398b570cb891b01260684ea537e4d4bc380ec).

Reply:
Thanks